tests/console/download_asset.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Downloads ASSET_1 file to specified by test_data file location.
```

Comment:
s/to specified by/specified/

Reply:
I understand it was a bit confusing without the commas.